src/commands/CommandMusicPlay.java

```diff
@@ -35,7 +48,30 @@ public void action(){
 			}
 
 			if(getContent() != null){
-				MusicManager.get().loadTrack(this, getContent());
+				String id;
+				String source;
+				if (!isUrl(getContent())){
+					YouTube youtube = new YouTube.Builder(new NetHttpTransport(), new JacksonFactory(), new HttpRequestInitializer() {
+						public void initialize(HttpRequest request) throws IOException {
+						}
+					}).setApplicationName("Discord Bot").build();
+					try {
+						YouTube.Search.List search = youtube.search().list("snippet");
+						search.setMaxResults((long) 1);
+						search.setQ(getContent());
```

Comment:
Give some space to your code! All lines after lines makes it harder to read, you can write some lines one after the other when their logic is similar, but all of them one after the other? Boï :)

My take would look like this :
```java
[...]
		// Yes, I'd also add an empty line before the " try{ "
try {

	YouTube.Search.List search = youtube.search().list("snippet");

	search.setMaxResults((long) 1);
	search.setQ(getContent());
	search.setKey(Environment.getVar("YOUTUBE_TOKEN"));

	SearchListResponse response = search.execute();

	id = response.getItems().get(0).getId().getVideoId();
	source = "https://www.youtube.com/watch?v=" + id;

	MusicManager.get().loadTrack(this, source);

} catch (IOException e) {
[...]
```